packages/data/src/registry.js

```diff
@@ -104,13 +104,8 @@ export function toAsyncIterable( object ) {
 			object = [ object ];
```

Comment:
Nitpick: The comment preceding the condition reads as a continuation which was previously continued as "...of Promises", now removed. This could be rewritten as a plain "Normalize as iterable"